test/test_other.py

```diff
@@ -13883,9 +13883,8 @@ def test_pthread_trap(self):
   def test_pthread_kill(self):
     self.do_run_in_out_file_test('pthread/test_pthread_kill.c')
 
-  @node_pthreads
   def test_emscripten_set_interval(self):
-    self.do_runf('emscripten_set_interval.c', args=['-pthread', '-sPROXY_TO_PTHREAD'])
+    self.do_runf('emscripten_set_interval.c')
```

Comment:
Previously this was tested with pthreads, but now it isn't, if I read this diff correctly?

Reply:
I see. Should we fix the typo and test it with pthreads, or not?

Reply:
I can't think of a reason to use pthreads on this myself. lgtm either way.

Reply:
We test with and without threads in the browser already.